src/mcedit2/panels/worldmeta.py

```diff
@@ -0,0 +1,255 @@
+"""
+    player
+"""
+from __future__ import absolute_import
+import logging
+from contextlib import contextmanager
+
+from PySide import QtGui
+from PySide.QtCore import Qt
+
+from mcedit2.util.resources import resourcePath
+from mcedit2.util.load_ui import load_ui
+from mcedit2.util.screen import centerWidgetInScreen
+from mcedit2.widgets.nbttree.nbteditor import NBTDataChangeCommand
+
+log = logging.getLogger(__name__)
+
+
+class WorldMetaPanel(QtGui.QWidget):
+    GENERATOR_TYPES = ["default", "flat", "largeBiomes", "amplified", "customized", 'debug_all_block_states']
+    GAMEMODES = ["Survival", "Creative", "Adventure", "Spectator"]
```

Comment:
These will need to be translated. Create them in `__init__` and pass each string to `self.tr()` individually.